x-pack/legacy/plugins/lens/public/indexpattern_plugin/operations/definitions/date_histogram.tsx

```diff
@@ -208,7 +208,11 @@ export const dateHistogramOperation: OperationDefinition<DateHistogramIndexPatte
                   <EuiFlexItem>
                     <EuiFieldNumber
                       data-test-subj="lensDateHistogramValue"
-                      value={interval.value}
+                      value={
+                        typeof interval.value === 'number' || interval.value === ''
```

Comment:
Not super problematic but seems odd that EUI accepts a number or an empty string... (Am I reading that right?)

I'd expect a type signature more like `number | undefined` maybe? 

Also, does this need a `NaN` check?